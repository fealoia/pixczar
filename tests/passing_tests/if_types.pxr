Void main() {
  Boolean x, y;
  x = true;
  y = true;
  if(x == y) {
    print("true expression");
  }
  if(x != y) {
    print("false expression");
  }
  if(1){
    print("Int");
  }
  if(1.0){
    print("Float");
  }
  if("s"){
    print("String");
  }
  if(new Pix()){
    print("Pix");
  }
  if(null){
    print("null");
  }
}
