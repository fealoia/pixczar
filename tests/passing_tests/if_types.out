true expression
Int
Float
String
Pix
